framework/PostProcessors/DataMining.py

```diff
@@ -579,6 +578,74 @@ def collectOutput(self, finishedJob, outputObject):
                                                        self.name, "Model",
                                                        missingKeys))
 
+  def collectOutput(self, finishedJob, outputObject):
+    """
+      Function to place all of the computed data into the output object
+      @ In, finishedJob, JobHandler External or Internal instance, A JobHandler
+        object that is in charge of running this post-processor
+      @ InOut, outputObject, dataObjects, A reference to an object where we want
+        to place our computed results
+    """
+    ## When does this actually happen?
+    evaluation = finishedJob.getEvaluation()
+    if isinstance(evaluation, Runners.Error):
+      self.raiseAnError(RuntimeError, "No available output to collect (run possibly not finished yet)")
+
+    inputObject, dataMineDict = evaluation
+
+    ## This should not have to be a list
+    ## TODO: figure out if there is a case where it can be in this processor
+    inputObject = inputObject[0]
+
+    ## Store everything we cannot wrangle from the input data object and the
+    ## result of the dataMineDict
+    missingKeys = []
+
+    ############################################################################
+    ## If the input data object is different from the output data object, we
+    ## need to explicitly copy everything over that the user requests from the
+    ## input object
+    if inputObject != outputObject:
+      ## First copy any data you need from the input object
+      ## before adding new data
+      numberOfRealizations =  len(inputObject['data']['RAVEN_sample_ID'].values)
+      for i in range(numberOfRealizations):
+        outputObject.addRealization(inputObject.realization(index=i))
+    ## End data copy
+    ############################################################################
+
+    ############################################################################
+    ## Now augment the DataObject by adding the computed labels and/or
+    ## transformed coordinates aka the new columns added to the DataObject
+    for key in dataMineDict['outputs']:
+      if outputObject.type == 'PointSet':
+        outputObject.addVariable(key,dataMineDict['outputs'][key])
+      elif outputObject.type == 'HistorySet':
```

Comment:
The following calculations seems to me are using the functions from the old DataObjects, such as the method 
```updateOutputValue```

Reply:
I dont think DataMining.py has been updated yet

Reply:
Good.